src/roles/nuh-deploy/tasks/nuh_copy_certificates.yml

```diff
@@ -34,25 +37,39 @@
         private_key: "/root/.ssh/proxypeer"
       delegate_to: "{{ groups['nuhs'][1] }}"
       run_once: true
-      when: groups['nuhs'] | length > 1
+      when: 
```

Comment:
This task (Copy proxy certificates...) can be removed. The NUH will take care of this for us.

Reply:
Done.

Reply:
Done.